src/helpers.js

```diff
@@ -90,6 +90,7 @@ module.exports.logItemCompletion = (successText) => {
 module.exports.logConclusion = () => {
   console.info('\n');
   console.info(chalk.bold.rgb(...colors.green)('Component created! 🚀 '));
+  console.info(chalk.rgb(...colors.blue)(`If this was a mistake, run ${chalk.bold`new-component-undo`}`));
```

Comment:
Can we change this copy to "Made a mistake? Undo this component by running 'new-component-undo'."

Reply:
I like it!